src/actions/Users.js

```diff
@@ -188,6 +190,19 @@ export function editUser(params) {
   };
 }
 
+export function updateForceReset(ud, successcb, errcb) {
+  ud.spec.forceReset = false;
```

Comment:
we dont need to send user as input

Reply:
As user information is already being passed via cookies, removed the above.